### ISSUE TITLE
Facing a issue when running on app.

### ISSUE THREAD

**Issue by yogesh-choudhary3** [mobile-dev-inc/maestro#178]
Yaml file
```yaml
appId: com.flipkart.android
clearState: true
---
- launchApp
- tapOn: "English"
- tapOn: "Continue"
```

Error stack

```
io.grpc.StatusRuntimeException: UNKNOWN        
        at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:262)
        at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:243)
        at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:156)
        at maestro_android.MaestroDriverGrpc$MaestroDriverBlockingStub.viewHierarchy(MaestroDriverGrpc.java:270)
        at maestro.drivers.AndroidDriver.contentDescriptor(AndroidDriver.kt:205)
        at maestro.ViewHierarchy$Companion.from-c1iYVAs(ViewHierarchy.kt:28)
        at maestro.Maestro.viewHierarchy-prqvCes(Maestro.kt:228)
        at maestro.orchestra.Orchestra.findElement(Orchestra.kt:257)
        at maestro.orchestra.Orchestra.findElement$default(Orchestra.kt:243)
	at maestro.orchestra.Orchestra.tapOnElement(Orchestra.kt:216)
	at maestro.orchestra.Orchestra.executeCommand(Orchestra.kt:120)
	at maestro.orchestra.Orchestra.executeCommands(Orchestra.kt:107)
	at maestro.orchestra.Orchestra.runFlow(Orchestra.kt:69)
	at maestro.cli.runner.MaestroCommandRunner.runCommands(MaestroCommandRunner.kt:98)
	at maestro.cli.runner.TestRunner$runSingle$result$1.invoke(TestRunner.kt:32)
	at maestro.cli.runner.TestRunner$runSingle$result$1.invoke(TestRunner.kt:29)
	at maestro.cli.runner.TestRunner.runCatching(TestRunner.kt:116)
	at maestro.cli.runner.TestRunner.runSingle(TestRunner.kt:29)
	at maestro.cli.command.TestCommand.call(TestCommand.kt:60)
	at maestro.cli.command.TestCommand.call(TestCommand.kt:30)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
	at picocli.CommandLine.access$1200(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
	at picocli.CommandLine.execute(CommandLine.java:2058)
	at maestro.cli.AppKt.main(App.kt:74)
```

**Comment by lalon** [comment]
Try adding "- hideKeyboard" before the step that fails as a temporary workaround.
I assume this issue is caused by long loading time.

**Comment by teresch-a** [comment]
> Try adding "- hideKeyboard" before the step that fails as a temporary workaround. I assume this issue is caused by long loading time.

I seem to have stack overflow because of it running with **maestro test -c flow.yaml**
<img width="593" alt="image" src="https://user-images.githubusercontent.com/79324308/190622498-9aa1820b-2927-4dc0-9c39-c232527c40fc.png">

**Comment by yogesh-choudhary3** [comment]
Yes, it worked Thanks @lalon @teresch-a

**Comment by frichez** [comment]
We are facing the same issue due to the fact that our android app is very slow to start (ok we have to work on this ;-)
In the meantime adding hideKeyboard command let the test pass. I didn't notice this on v1.5.

**Comment by dmitry-zaitsev** [comment]
I believe this should be fixed in 1.7.0. Please let me know if that is not the case as we were still not able to reproduce it

**Comment by sebastinto** [comment]
> I believe this should be fixed in 1.7.0. Please let me know if that is not the case as we were still not able to reproduce it

I'm experiencing the issue using 1.8.0. Adding `- hideKeyboard` right after the `- launchApp` fixes it.
